src/antsibull_docs/cli/doc_commands/_build.py

```diff
@@ -0,0 +1,217 @@
+# Author: Toshio Kuratomi <tkuratom@redhat.com>
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or
+# https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+# SPDX-FileCopyrightText: 2020, Ansible Project
+"""Utilities for various docs build subcommands."""
+
+import textwrap
+import typing as t
+
+from antsibull_core.compat import asyncio_run
+from antsibull_core.logging import log
+from antsibull_core.venv import FakeVenvRunner, VenvRunner
+
+from ... import app_context
```

Comment:
I find these deep relative imports very confusing. IMO, using the absolute name would be better.

Reply:
Hmm, I personally like that we always use relative imports for things in the same antsibull* package.